bedrock/security/utils.py

```diff
@@ -72,12 +72,15 @@ def parse_yml_file(file_name):
         if mfsa_id:
             data['mfsa_id'] = mfsa_id
 
-    return data, generate_yml_advisories_html(data['advisories'])
+    return data, generate_yml_advisories_html(data)
 
 
-def generate_yml_advisories_html(advisories):
+def generate_yml_advisories_html(data):
     html = []
-    for cve, advisory in advisories.items():
+    if 'description' in data:
+        html.append(markdown(data['description']))
+
+    for cve, advisory in data['advisories'].iteritems():
```

Comment:
Yeah. I'm always torn now with these. Should I get the efficiency now with Python 2, or make the transition to 3 easier for later. I usually opt for the former because the 2to3 tool is pretty good at that kinda thing, but it is a concern. For bedrock I think python3 is still a long way off, but hopefully I'm wrong about that.